app/controllers/users_controller.rb

```diff
@@ -1,69 +0,0 @@
-class UsersController < ApplicationController
-  before_action :set_user, only: %i[show edit update destroy]
```

Comment:
- [ ] Kindly restore this controller, as it is the correct controller you should have. Kindly keep only the `index` and `show` actions.